packages/dashboard/src/layout/LayoutUtils.ts

```diff
@@ -106,17 +105,20 @@ class LayoutUtils {
    * @returns The newly created stack.
    */
   static addStack(parent: ContentItem, columnPreferred = true): Stack {
-    const type = columnPreferred ? 'column' : 'row';
+    const rowOrColConfig: ItemConfig = {
+      type: columnPreferred ? 'column' : 'row',
+    };
+
```

Comment:
Move this to within the `if (isRoot(parent))` block below, which is the only place `rowOrColConfig` is used.

Reply:
fixed